cibyl/plugins/openstack/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+API_CLASS = {"deployment": "Deployment",
```

Comment:
We don't really need to add all models. Just adding Deployment is enough, since the rest of models (Node, Service, Container) are part of its API

Reply:
+ We probably don't want to manually maintain such dictionaries. Just make the link between CI models and plugin models

Reply:
@bregman-arie Link means what i understand is calling plugin from CI models.